### PR TITLE
Update issue and pr template

### PR DESCRIPTION
**Pull request by qwq233**
Signed-off-by: qwq233 <qwq233@qwq2333.top>

Update issue and pr template

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app's APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
+# Automatically convert third-party libraries to use AndroidX
+android.enableJetifier=true
+--warning-mode=all
+org.gradle.java.home=C:/Program Files/Java/jdk-14.0.2
```
Comment (keta1) [inline gradle.properties:21]: ^M啥玩意

Comment (qwq233) [reply gradle.properties]: 呜呜呜无视就好 貌似是crlf的问题 windows环境就是麻烦(碎碎念)

**Comment by keta1** [comment]
好多啊，不想看了，你们慢慢看

**Comment by qwq233** [comment]
> 好多啊，不想看了，你们慢慢看

刷新下= =

**Review by singleNeuron — APPROVED** [review]
记着skip ci

**Review by NextAlone — APPROVED** [review]

**Pull request merged**